src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/SetAzureRmDataLakeStoreItemAclEntry.cs

```diff
@@ -79,8 +80,21 @@ public class SetAzureDataLakeStoreItemAclEntry : DataLakeStoreFileSystemCmdletBa
             Mandatory = false, HelpMessage = "Indicates that the ACL entry is a default ACE to be set.")]
         public SwitchParameter Default { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Indicates the resulting ACL should be returned.",
+            ParameterSetName = BaseParameterSetName
+            )]
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Indicates the resulting ACL should be returned.",
+            ParameterSetName = SpecificAceParameterSetName
+            )]
```

Comment:
@begoldsm this parameter does not need any parameterset names. This parameter will be part of global parameter set